client/src/components/Navbar.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+import Dropdown from './DropdownMenu/Dropdown';
+
+const people = [
+  { name: 'Edmon Wong', link: 'https://github.com/Edmond120' },
+  { name: 'Aziz Yokubjonov', link: 'https://github.com/aziz512' },
+  { name: 'Haitham Alnajar', link: 'https://www.google.com' },
+  { name: 'Yankang (Kyle) Xue', link: 'https://github.com/KanG98' },
+  { name: 'Kamille Tipan', link: 'https://github.com/aziz512/youtube-tracker' },
+];
+
+const Navbar = () => {
+  return (
+    <Nav>
+      <Logo href="">
+        Video<span>Tube</span>
+      </Logo>
+      <Menu>
+        <Dropdown people={people}></Dropdown>
+        <MenuLink
+          href="https://github.com/aziz512/youtube-tracker"
+          target="_blank"
+        >
+          Our Work
+        </MenuLink>
+      </Menu>
+    </Nav>
+  );
+};
+
+export default Navbar;
+
+const MenuLink = styled.a`
+  padding: 1rem 2rem;
+  cursor: pointer;
+  text-align: center;
+  text-decoration: none;
+  color: #67bc98;
+  transition: all 0.3s ease-in;
+  font-size: 0.9rem;
```

Comment:
It's sort of hard to read right now, maybe make it bold?

Reply:
Increase font-weight and it seems to be easier to read now.